ftl-build/armv6hf/Dockerfile

```diff
@@ -52,9 +52,9 @@ RUN mkdir rpi_tools && cd rpi_tools && git init && git remote add origin https:/
     cp -a arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/* /usr/ && rm -rf ../rpi_tools
 
 # Download procompiled armv6 libraries
-RUN wget ftl.pi-hole.net/libraries/libgmp.a -O /usr/local/lib/libgmp.a && \
-    wget ftl.pi-hole.net/libraries/libnettle.a -O /usr/local/lib/libnettle.a && \
-    wget ftl.pi-hole.net/libraries/libhogweed.a -O /usr/local/lib/libhogweed.a
+RUN wget https://ftl.pi-hole.net/libraries/libgmp.a -O /usr/local/lib/libgmp.a && \
+    wget https://ftl.pi-hole.net/libraries/libnettle.a -O /usr/local/lib/libnettle.a && \
+    wget https://ftl.pi-hole.net/libraries/libhogweed.a -O /usr/local/lib/libhogweed.a
```

Comment:
These are small files, we can probably just include them in version control (`git`) and pull from the GitHub API instead of from a server file storage location.

Reply:
`libhogweed.a` is 9MB alone, the others are around 1MB each. They are needed really only when new base images are built and it seems better to just download them when we need them.

Reply:
And as far as I know, they're only in the armv6 base image, right? The rest don't have a reference to these files